views/unsupported.ejs

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="<%= language.code %>">
+  <head>
+    <meta charset="utf-8">
+    <title><%= _('Qwant Maps') %> - <%= _('Unsupported browser', 'unsupported') %></title>
+    <meta name="description" content="<%= _('The map that respects your privacy') %>">
+    <meta name='viewport' content="width=device-width,initial-scale=1"/>
+    <link href='./statics/css/unsupported.css' rel='stylesheet'/>
+    <link href='./mapstyle/iconfont/iconfont.css' rel='stylesheet'/>
```

Comment:
Is this `iconfont.css` really required ?

Reply:
No indeed, thanks for spotting it :+1: